fsspec/implementations/tests/test_local.py

```diff
@@ -577,8 +582,8 @@ def test_linked_directories(tmpdir):
 
     fs = LocalFileSystem()
     assert fs.info(subdir0)["type"] == "directory"
-    assert fs.info(subdir1)["type"] == "directory"
-    assert fs.info(subdir2)["type"] == "directory"
```

Comment:
This is also a desired information though. Link is an fs implementation detail, so users should be able to tell that this is a directory first and also that this is a link. So it looks like it should stay as `type==directory` here and so `islink` is not actually obsoleted.